### ISSUE TITLE
install fluxgym  and start error

### ISSUE THREAD

**Issue by gushenjie** [cocktailpeanut/fluxgym#319]
![Image](https://github.com/user-attachments/assets/be03e6e2-8f58-404e-9aa8-de3cdcd7c8cb)

Follow your tutorial to install fluxgym and report an error when starting.

**Comment by gushenjie** [comment]
python version 3.11.9

**Comment by gushenjie** [comment]
[2025-01-31 13:25:28] [INFO] num reg images / 正則化画像の数: 0
[2025-01-31 13:25:28] [INFO] num batches per epoch / 1epochのバッチ数: 40
[2025-01-31 13:25:28] [INFO] num epochs / epoch数: 16
[2025-01-31 13:25:28] [INFO] batch size per device / バッチサイズ: 1
[2025-01-31 13:25:28] [INFO] gradient accumulation steps / 勾配を合計するステップ数 = 1
[2025-01-31 13:25:28] [INFO] total optimization steps / 学習ステップ数: 640
[2025-01-31 13:25:47] [INFO] steps:   0%|          | 0/640 [00:00<?, ?it/s]2025-01-31 13:25:47 INFO     unet dtype:                  train_network.py:1264
[2025-01-31 13:25:47] [INFO] torch.float8_e4m3fn, device:
[2025-01-31 13:25:47] [INFO] cuda:0
[2025-01-31 13:25:47] [INFO] INFO     text_encoder [0] dtype:      train_network.py:1270
[2025-01-31 13:25:47] [INFO] torch.float8_e4m3fn, device:
[2025-01-31 13:25:47] [INFO] cuda:0
[2025-01-31 13:25:47] [INFO] INFO     text_encoder [1] dtype:      train_network.py:1270
[2025-01-31 13:25:47] [INFO] torch.bfloat16, device: cpu
[2025-01-31 13:25:47] [INFO] 
[2025-01-31 13:25:47] [INFO] epoch 1/16
[2025-01-31 13:25:47] [INFO] 
[2025-01-31 13:25:53] [INFO] 2025-01-31 13:25:53 INFO     epoch is incremented.            train_util.py:764
[2025-01-31 13:25:53] [INFO] current_epoch: 0, epoch: 1
[2025-01-31 13:25:53] [INFO] 2025-01-31 13:25:53 INFO     epoch is incremented.            train_util.py:764
[2025-01-31 13:25:53] [INFO] current_epoch: 0, epoch: 1


torch.bfloat16, device: cpu, does this affect the training speed? How can I change it to gpu? My device is Nvidia 3090 24GB graphics card

**Comment by OKSoda** [comment]
I have this same problem
